### PR TITLE
Add media gallery and new screenshots

### PR DESCRIPTION
**Pull request by karolkielecki**

### CODE REVIEW

**Review by maarcingebala — APPROVED** [review]

File: docs/dashboard/catalog/products.md [hunk 1]
```diff
@@ -229,22 +229,29 @@ To show a product in the product listings, you have to enable the _Show in produ
...
 
 - Drag and drop a file from a folder on your computer
-- Click on _Upload&nbsp;Image_ in the top right corner and select an image
+- Click the _Upload_ button in the top right corner of the media section and choose _Upload Images_. The next step is to select images from your computer
 
-You can add an unlimited number of images per product. The main image will be the one that is first in line. You can drag and drop images to change the order. Hover over a specific image and click the trash can to delete an unwanted image.
+Saleor also allows users to upload videos through URL links. To upload a video, follow the steps below:
 
-![Uploading product images](../screenshots/catalog-product-image-upload.jpeg)
+- Publish a video on a service such as YouTube
```
Comment (krzysztofwolski) [inline docs/dashboard/catalog/products.md:241]: Which else services do we support? There should be full list

**Review by krzysztofwolski — CHANGES_REQUESTED** [review]

File: docs/dashboard/catalog/products.md (hunk 1 above)
Comment (karolkielecki) [reply docs/dashboard/catalog/products.md:241]: @maarcingebala What else can we add here?

Comment (maarcingebala) [reply docs/dashboard/catalog/products.md:241]: Dailymotion, Flickr, Imgur, Instagram, MobyPicture, Photobucket, SmugMug, Scribd, TikTok, Twitter, Vimeo, Wordpress.tv, YouTube

Comment (maarcingebala) [reply docs/dashboard/catalog/products.md:241]: This is the full list of services that our validation library supports.

**Review by krzysztofwolski — APPROVED** [review]
When editing documents and uploading

**Pull request merged**